gsheet2notion.py

```diff
@@ -3,92 +3,158 @@
 # SPDX-FileCopyrightText: 2020 IgorKha <hahabahov@yandex.ru>
 # SPDX-License-Identifier: GPL-3.0
 #
-# Send updates from Mesto's Google Sheet to Notion
-#
-#
+
+"""Send updates from Mesto's Google Sheet to Notion."""
 
 # Standard imports
 import csv
+from typing import Final, List
 
 # Third party packages
 import click
+import gspread
 import pandas as pd
 from notion.client import NotionClient
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
 
 # Project specific imports
-from gsh import googlesheet
-from secret import db_file, gsheet_page, notion_key, notion_url, slack_channel
-from slack_msg import message
+from secret import (
+    db_file,
+    gsheet_page,
+    gsheet_url,
+    notion_key,
+    notion_url,
+    slack_channel,
+    slack_key
+)
+
+# BEGIN Constants
+_COLUMNS: Final[List[str]] = [
+    'Date'
+  , 'Name'
+  , 'Email'
+  , 'OpenLand'
+  , 'Telegram'
+  , 'Team_to_help'
+  , 'Competence'
+  , 'Task'
+  , 'Profit'
+  , 'Why_we'
+  , 'work_hour'
+  , 'timezone'
+  , 'About_me'
+  , 'otkuda_uznal'
+  , 'CV'
+  , 'first_status'
+  , 'Interview'
+  , 'Result'
+  ]
+# END Constants
+
+
+# BEGIN Globals
+# TODO Get rid of the globals!
+# class slackmessage:
+wclient = WebClient(token=slack_key)
+# END Globals
+
+
+# BEGIN Internal functions
+def _read_google_sheet(sheet):
+    """ Read google sheet. """
+    gc = gspread.service_account(filename='credentials.json')
+    sh = gc.open_by_url(gsheet_url)                         # Open the file by url
+    worksheet = sh.worksheet(sheet)                         # Select the sheet inside the file
+    return worksheet.get_all_records()
+
+
+def _send_message_to_slack(chat, text):
+    try:
+        response = wclient.chat_postMessage(channel=chat, text=text)
+        assert response['message']['text'] == text
+
+    except SlackApiError as e:
+        # You will get a SlackApiError if 'ok' is False
+        assert e.response['ok'] is False
+        assert e.response['error']  # str like 'invalid_auth', 'channel_not_found'
+        print(f'Got an error: {e.response["error"]}')
+        # TODO Handle the exception properly!
+# END Internal functions
 
 
 @click.command()
 @click.help_option(
     '--help'
-    , '-h'
-)
+  , '-h'
+  )
 @click.version_option(message='%(version)s')
 def gsheet2notion():
-    client = NotionClient(token_v2=notion_key)
+    """Send updates from Mesto's Google Sheet to Notion."""
```

Comment:
@IgorKha 

Do not remove this docstring! It's used as the program description for `--help` option.